app/src/main/res/values/localized.xml

```diff
@@ -95,6 +95,7 @@
     <string name="p_runs">Runs</string>
     <string name="p_roll_limit">Limit number of rolls</string>
     <string name="p_prevent_lottery_box_reset">Prevent box reset</string>
+    <string name="p_receive_embers">Receive embers</string>
```

Comment:
Please translate p_receive_embers, stop_after_this_run, tapper_service_description, overlay_notification_text and p_roll_limit.

Reply:
Done

Reply:
Make some words I used in last commit more formal and fixed a typo (again).

Reply:
tapper_service_description is still missing.